### ISSUE TITLE
does this work for the newly released Gimp 2.10 ?

### ISSUE THREAD

**Issue by luzpaz** [cttynul/gimpshop-reloaded#4]
and the upcoming GTK3 branch ?

**Comment by luzpaz** [comment]
bump

**Comment by gatlinnewhouse** [comment]
Yes but you need to install it to the folder: `~/.config/GIMP/2.10`

**Comment by gatlinnewhouse** [comment]
There are some bugs if you set your theme to `NEW CS6 II theme for gimp` in GIMP's preferences, but otherwise everything seems to be working (note: I haven't done extensive testing)

**Comment by gatlinnewhouse** [comment]
[This](https://github.com/AbdullahRagb/gimp-dark-theme) might be a better theme but that's just going off of when it was recently updated.

**Comment by cttynul** [comment]
It works with Gimp 2.10 a part graphical theme, it's the only thing a bit broken.

If you find some theme that works like a charm with GIMP 2.10 link that here, please.
